File: dataset/codenet/Python/Code/s067092519.py
N = int(input())
m=[]
line = ''
m = [[[0 for k in range(10)] for j in range(3)] for i in range(10)]
for i in range(N):
    n = input().split()
    b = int(n[0])
    f = int(n[1])
    r = int(n[2])
    v = int(n[3])
    m[b-1][f-1][r-1] += v

for i in range(4):
    for j in range(3):
        for k in range(10):
            line += ' {0}'.format(m[i][j][k])
        print(line)
        line = ''
    if i < 3:
        print('#'*20)

File: output/gpt_4o/codenet/algo_based_translation/python/java/successful_translation/s067092519.java

import java.util.Scanner;

public class s067092519 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int N = Integer.parseInt(scanner.nextLine());
        
        int[][][] m = new int[10][3][10];
        
        for (int i = 0; i < N; i++) {
            String[] n = scanner.nextLine().split(" ");
            int b = Integer.parseInt(n[0]);
            int f = Integer.parseInt(n[1]);
            int r = Integer.parseInt(n[2]);
            int v = Integer.parseInt(n[3]);
            m[b-1][f-1][r-1] += v;
        }
        
        for (int i = 0; i < 4; i++) {
            for (int j = 0; j < 3; j++) {
                StringBuilder line = new StringBuilder();
                for (int k = 0; k < 10; k++) {
                    line.append(" ").append(m[i][j][k]);
                }
                System.out.println(line.toString());
            }
            if (i < 3) {
                System.out.println("#".repeat(20));
            }
        }
        
        scanner.close();
    }
}

